### PR TITLE
added atera scanner

### PR DESCRIPTION
**Pull request by ladybug0125**

### CODE REVIEW

File: pkg/detectors/atera/atera.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			Raw:          []byte(resMatch),
+		}
+
+		if verify {
+			req, err := http.NewRequestWithContext(ctx, "GET", "https://app.atera.com/api/v3/alerts", nil)
+			if err != nil {
+				continue
+			}
+			req.Header.Add("Accept", "application/json")
+			req.Header.Add("X-API-KEY", fmt.Sprintf("%s", resMatch))
```
Comment (ahrav) [inline pkg/detectors/atera/atera.go:56]: nit: you don't need to use `fmt.Sprintf` here. You can just use `resMatch`.

File: pkg/detectors/atera/atera_test.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "found, unverified",
+			s:    Scanner{},
+			args: args{
+				ctx:    context.Background(),
+				data:   []byte(fmt.Sprintf("You can find a atera secret %s within but not valid", inactiveSecret)), // the secret would satisfy the regex but not pass validation
```
Comment (ahrav) [inline pkg/detectors/atera/atera_test.go:59]: nit: find `an` vs find `a`

File: pkg/detectors/atera/atera_test.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		args    args
+		want    []detectors.Result
+		wantErr bool
+	}{
+		{
+			name: "found, verified",
+			s:    Scanner{},
+			args: args{
+				ctx:    context.Background(),
+				data:   []byte(fmt.Sprintf("You can find a atera secret %s within", secret)),
```
Comment (ahrav) [inline pkg/detectors/atera/atera_test.go:43]: nit: same here `an` vs `a`

**Review by ahrav — APPROVED** [review]
Small nitpicks otherwise lgtm

File: pkg/detectors/atera/atera_test.go (hunk 2 above)
Comment (ladybug0125) [reply pkg/detectors/atera/atera_test.go]: Thanks! I just pushed the changes.

File: pkg/detectors/atera/atera.go (hunk 1 above)
Comment (ladybug0125) [reply pkg/detectors/atera/atera.go]: Thanks! I just pushed the changes.

File: pkg/detectors/atera/atera_test.go (hunk 3 above)
Comment (ladybug0125) [reply pkg/detectors/atera/atera_test.go]: Thanks! I just pushed the changes.

**Pull request merged**